README.md

```diff
@@ -1,58 +1,90 @@
 
 ## Motivation
 
-{package-motivation}
+Truncating a text and appending an ellipsis (`...`) to the end of it is a very common use case. It's pretty straightforward to implement, by using CSS only, if we're talking single line. However, for multi-line texts, things get a little bit trickier. Even trickier when you don't really know how many lines of text you're going to have, as it all depends on the css properties you will set for the `<div>` your text will be in. This package aims to ease the pain of appending an ellipsis to any text, while also taking care of responsivity.
 
 ## Usage
 
-{package-usage-example}
+In the component where the truncated text is meant to be placed, import the `Ellipsis` component and just send the text as its child, in `string` format. Like so:
+
+```js
+import React from 'react';
+import Ellipsis from '@moxy/react-ellipsis';
+
+import styles from '...';
+
+const Home = () => (
+    <div>
+        <h1>react-ellipsis</h1>
+        <Ellipsis className={ styles.text }>
+            Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae est. Mauris placerat eleifend leo. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae est. Mauris placerat eleifend leo. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae est. Mauris placerat eleifend leo.
+        </Ellipsis>
+    </div>
+);
+```
+
+It works just fine if you pass a height and/or width as a style to the `Ellipsis`. The text will then appear truncated, with an ellipsis appended at the end.
+
 
 ## API
 
-{package-api-description}
+These are the props available in `@moxy/react-ellipsis`.
 
-#### {package-api-prop-example}
+#### children
 
-Type: `object`
+Type: `string`
 Required: `true`
 
-The `{package-api-prop-example}` has the following shape:
-```js
-{package-api-prop-example}: PropTypes.shape({
-    foo: PropTypes.string,
-    bar: PropTypes.arrayOf(PropTypes.object),
-}).isRequired,
-```
+The string that will be truncated by the `Ellipsis` component.
+
+#### tag
+
+Type: `string`
+
+The tag that will wrap the truncated text Default is `p`.
+
+#### className
+
+Type: `string`
+
+A className to apply to the component.
+
+#### onMouseEnter
+
+Type: `func`
+
+A callback that will be called when the mouse pointer enters the contents of the `Ellipsis` component.
+
+#### onMouseLeave
+
+Type: `func`
+
+A callback that will be called when the mouse pointer leaves the contents of the `Ellipsis` component.
 
```

Comment:
As there is no mention of it, I assume no props are being spread, so we should add the `onClick` ability here too (and in the code).

Reply:
Props are actually being spread. `onMouseEnter` and `onMouseLeave` should probably be removed altogether and we metion in the docs that props are being spread, giving some examples like the ones mentioned. Thoughts?

Reply:
Yup, I commented on this before checking the code itself. 

> onMouseEnter and onMouseLeave should probably be removed altogether and we metion in the docs that props are being spread, giving some examples like the ones mentioned.

This is the way to go 👍